src/visitor.js

```diff
@@ -67,11 +67,21 @@ export default class Visitor {
       const newRule: Rule = postcss.rule({ selector: `--${setName}` });
 
       // $FlowFixMe
-      Object.entries(sets[setName]).forEach(([prop, value]) => {
-        newRule.prepend(
-          postcss.decl({ prop: kebabify(prop), value })
+      const set = sets[setName];
+
+      if (typeof set === 'string') {
+        newRule.prepend(set);
+      } else if (typeof set === 'object') {
```

Comment:
That's not an acceptable Object testing. 
`typeof [] --> 'object'`
`typeof null --> 'object'`